nucypher/blockchain/eth/clients.py

```diff
@@ -197,20 +197,20 @@ def coinbase(self) -> str:
         return self.w3.eth.coinbase
```

Comment:
I don't see any tests for a lot of the new methods in this file

Reply:
Which new methods?  I do see one: `new_account`.

Reply:
Yes that one, but I am pretty sure that some of these will fail to work in some of the various clients which will necessitate another `fix-ganache-issues` branch.